Jenkinsfile

```diff
@@ -107,12 +136,35 @@ make build-app OPS_URL=$OPS_URL"""
                     AWS_KEYPAIR=ops \
                     AWS_REGION=us-east-1 \
                     ROBOTEST_VERSION=$ROBOTEST_VERSION"""
-            }
-          }else {
-            echo 'skipped system tests'
+                }
+            } )
           }
-        } )
+        }
+      } else {
+        echo 'skipped system tests'
       }
     }
+
+    stage('import') {
+      if (params.IMPORT_APP) {
+        withCredentials([
+          string(credentialsId: params.OPS_CENTER_CREDENTIALS, variable: 'API_KEY'),
+        ]) {
+          withEnv(MAKE_ENV) {
+            sh 'make import'
```

Comment:
`make import` is working incorrectly here. I am going to add another make target to build app and provide you guidance on how to use that.

Reply:
Ok, I'll wait for the final target to do the changes in the other projects.